src/ansys/dpf/core/unit_system.py

```diff
@@ -95,34 +95,39 @@ def unit_names(self) -> str:
             return self._unit_names
 
 
-class unit_systems:
-    """Contains common Ansys predefined UnitSystems.
+try:
```

Comment:
@PProfizi This means that starting from now we will need to add `try/catch` blocks in all code that we add? Not talking about tests, just the source code. I think this will decrease the readability...

Reply:
@rafacanton indeed, I required this locally when running doctest specifically on this file. I guess doctest first tried to execute the file, which failed for 6.0. 
I'll try without.

Reply:
@cbellot000 @anslpa Just to let you know, after discussing this with Rafael, we decided that the right pattern is to put a "Required Version" note for the "helper" `unit_systems` class just like for the "feature" `UnitSystem` class, as well as a try-catch when populating it, so that the `unit_systems` class may appear for previous server versions, but it will be empty and the doc will indicate why.
Do you think this pattern is ok?

Reply:
@PProfizi, @cbellot000, @anslpa we could maybe stablish this kind of pattern for the future when this happens again